workshops/geometric_pattern/README.md

```diff
@@ -0,0 +1,538 @@
+# Geometric Pattern
+
+We'll be creating a cool graphic like the one above, using JavaScript and p5.js.
+[p5.js](http://p5js.org/) is a library for making stuff in conjunction with the HTML canvas element. It is nothing more than JS code written to make commonly-desired functionality much more accessible. It is a JS port of Processing, a software for making pictures and visuals with code. 
```

Comment:
This introduction needs work. The user probably knows very few, if any, of these terms yet.

Maybe we could use my sister as a gauge for what's already known going into this?